setup.py

```diff
@@ -24,6 +24,7 @@
         "netcdf3": ["scipy"],
     },
     entry_points={
+        "xarray.backends": ["kerchunk=kerchunk.xarray_backend:KerchunkBackend"],
```

Comment:
no need to add the entrypoint in both `setup.cfg` and `setup.py` (or rather, it's probably better not to do that).

As a side note, I'd also recommend transitioning to `pyproject.toml` since that's the most stable way going forward (but that would definitely have to be a separate PR).

Reply:
Would be happy to see a PR.